test/GSN/GSNBouncerSignature.test.js

```diff
@@ -17,58 +19,107 @@ contract('GSNBouncerSignature', function ([_, signer, other]) {
     });
   });
 
+  context('when constructor is called with a zero address', function () {
+    it('fails when constructor called with a zero address', async function () {
+      await expectRevert(
+        GSNBouncerSignatureMock.new(
+          '0x0000000000000000000000000000000000000000'
```

Comment:
You could use `ZERO_ADDRESS` here:

```
const { constants } = require('openzeppelin-test-helpers');
const { ZERO_ADDRESS } = constants;
```

Reply:
I did not know about that! Thats really useful!

Reply:
Added it.